docs/sources/panels/transformations/_index.md

```diff
@@ -23,13 +22,23 @@ The order in which transformations are applied can make a huge difference in how
 
 Before you can configure and apply transformations:
 
-- You must have entered a query and returned data from a data source. For more information on queries, refer to [Queries]({{< relref "../queries.md" >}}).
-
-- You must have chosen a visualization that supports your query result and chosen transforms. Different transformations can result in a data structure that only some visualizations can render. For example, Graph visualization cannot show reduced data (that results from the Reduce transform). If your current visualization does not show you what you expect after applying a transform, switch to the Table visualization to check the result. For more information on the different types of visualizations, refer to:
-  - [Bar gauge]({{< relref "../visualizations/bar-gauge-panel.md" >}})
-  - [Gauge]({{< relref "../visualizations/gauge-panel.md" >}})
-  - [Graph]({{< relref "../visualizations/graph-panel.md" >}})
-  - [Heatmap]({{< relref "../visualizations/heatmap.md" >}})
-  - [Logs]({{< relref "../visualizations/logs-panel.md" >}})
-  - [Stat]({{< relref "../visualizations/stat-panel.md" >}})
-  - [Table]({{< relref "../visualizations/table/_index.md" >}})
+- You must have entered a query that return data. For more information on queries, refer to [Queries]({{< relref "../queries.md" >}}).
+
+## List of transformations 
+
+- [Add field from calculation]({{< relref "./types-options.md#add-field-from-calculation" >}})
+- [Concatenate fields]({{< relref "./types-options.md#concatenate-fields" >}})
+- [Config from query results]({{< relref "./config-from-query.md" >}})
```

Comment:
@torkelo Should we start laying the foundation for our expanded "experimental" feature type by noting the status here? For example:

- [Config from query results]({{< relref "./config-from-query.md" >}}) *(Experimental)*

Reply:
Yea, not 100% these two transforms became a lot more solid than I originally expected. But could still be good to mark them as experimental, the threshold handling is especially so. thoughts @ryantxu ?